SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -282,7 +284,65 @@ void ASpatialDebugger::OnEntityAdded(const Worker_EntityId EntityId)
 		if (Actor->IsA<APlayerController>())
 		{
 			LocalPlayerController = Cast<APlayerController>(Actor);
+
+			if (ConfigUIWidget == nullptr)
+			{
+				FSoftClassPath ConfigUIClassRef(DEFAULT_CONFIG_UI_WIDGET_CLASS);
+				UClass* ConfigUIClass = ConfigUIClassRef.TryLoadClass<UUserWidget>();
+				if (ConfigUIClass)
+				{
+					ConfigUIWidget = CreateWidget<USpatialDebuggerConfigUI>(LocalPlayerController.Get(), ConfigUIClass);
+					ConfigUIWidget->SetSpatialDebugger(this);
+				}
```

Comment:
Should we print a warning if either the widget is unset, or this TryLoadClass fails?